escrow/payment_channel_storage.go

```diff
@@ -97,6 +97,14 @@ type BlockchainChannelReader struct {
 	recipientPaymentAddress   func() common.Address
 }
 
+func NewBlockChainChannelReader(a func(channelID *big.Int) (channel *blockchain.MultiPartyEscrowChannel, ok bool, err error), b func() common.Address) *BlockchainChannelReader {
```

Comment:
If I understand correct this method and `NewChannelPaymentValidatorT` are added to allow creating `BlockchainChannelReader` and `ChannelPaymentValidator` without real Ethereum RPC endpoint. But actually there is blockchain parameter `BlockchainEnabledKey` which disables connection to Ethereum RPC endpoint. And if you set this key to `false` you can use normal `blockchain.Processor` instance to pass it to usual constructors. So these two new constructors are not required actually and I would suggest removing them.

Reply:
Ok, actually I am not correct: 
- disabling `blockchain.Processor` on the configuration level will either lead to disabling payment infrastructure
- disabling `blockchain.Processor` in processor itself will lead to the payment check failures

So it looks like at the moment we cannot test channel closing behavior without `blockchain.Processor`.